ide/lsp.client/nbproject/project.properties

```diff
@@ -18,10 +18,10 @@
 javac.source=1.8
 javac.compilerargs=-Xlint -Xlint:-serial
 javadoc.arch=${basedir}/arch.xml
-release.external/org.eclipse.lsp4j-0.8.1-p1.jar=modules/ext/org.eclipse.lsp4j-0.8.1-p1.jar
-release.external/org.eclipse.lsp4j.generator-0.8.1.jar=modules/ext/org.eclipse.lsp4j.generator-0.8.1.jar
-release.external/org.eclipse.lsp4j.jsonrpc-0.8.1.jar=modules/ext/org.eclipse.lsp4j.jsonrpc-0.8.1.jar
-release.external/org.eclipse.xtend.lib-2.18.0.jar=modules/ext/org.eclipse.xtend.lib-2.18.0.jar
-release.external/org.eclipse.xtend.lib.macro-2.18.0.jar=modules/ext/org.eclipse.xtend.lib.macro-2.18.0.jar
-release.external/org.eclipse.xtext.xbase.lib-2.18.0.jar=modules/ext/org.eclipse.xtext.xbase.lib-2.18.0.jar
+release.external/org.eclipse.lsp4j-0.9.0.jar=modules/ext/org.eclipse.lsp4j-0.9.0.jar
```

Comment:
There are similar changes on my [java LSP debugging branch](https://github.com/JaroslavTulach/netbeans/tree/java-lsp-server-debugging-attempt2). Feel free to integrate soon. I will resolve the merge conflicts next week then.